xyz

```diff
@@ -141,32 +143,42 @@ read -r -s  # suppress user input
 echo        # output \n since [enter] output was suppressed
 
 run() {
-  echo "$1"
+  local arg
+  for arg ; do
+    if [[ $(printf "%q" "$arg") == "$arg" ]] ; then
+      printf "%s " "$arg"
+    else
+      printf "'%s' " "${arg//"'"/"'"'"'"'"'"'"'"}"
```

Comment:
@jonmarkprice, this is the amusing code we discussed a few days ago on Gitter. :)

Reply:
Ah nice 😄